README.md

```diff
@@ -1,6 +1,6 @@
-# ⛔️DEPRECATED
+# ⛔️This project is no longer supported and will be permanently deleted in 1 January 2021.
```

Comment:
trivial: "on the 1st of January 2021" sounds better

Reply:
done